target/hexagon/idef-parser/parser-helpers.c

```diff
@@ -80,36 +80,43 @@ bool is_inside_ternary(Context *c)
 /* Print functions */
 void str_print(Context *c, YYLTYPE *locp, const char *string)
 {
+	(void)locp;
```

Comment:
Mh, do any of the `_print` method ever uses `locp`? If not, let's drop the arguments everywhere.

Reply:
Mind the whitespaces.
```suggestion
	  (void) locp;
```

Reply:
There are some print functions that use `locp`, namely `rvalue_out`, `imm_out`, and `reg_arg_print` so we can't drop it sadly. I've also gone ahead and made `reg_arg_print` static as it is not exposed in the header.

I have changed my editor settings to use 4 spaces as per the qemu coding standard so that should fix most indent problems. To clarify, should `(void) locp` be indented two spaces relative to the rest of the code? Haven't seen that before.

Reply:
> There are some print functions that use `locp`, namely `rvalue_out`, `imm_out`, and `reg_arg_print` so we can't drop it sadly. I've also gone ahead and made `reg_arg_print` static as it is not exposed in the header.

ACK.

> I have changed my editor settings to use 4 spaces as per the qemu coding standard so that should fix most indent problems. To clarify, should `(void) locp` be indented two spaces relative to the rest of the code? Haven't seen that before.

Uhm, no, just like any other line. But here it was indented less then the next line.